packages/jest-reporters/src/notify_reporter.ts

```diff
@@ -118,6 +118,7 @@ export default class NotifyReporter extends BaseReporter {
             closeLabel: 'Close',
             icon,
             message,
+            timeout: 10,
```

Comment:
is this in seconds?

Reply:
Yep, it is, per their readme: https://github.com/mikaelbr/node-notifier

Good question though! My ideal API would have specified the unit e.g. `timeoutS` or `timeoutMS`.